doc/tutorials/using_zephyr_as_uos.rst

```diff
@@ -0,0 +1,88 @@
+.. _using_zephyr_as_uos:
+
+Using Zephyr as User OS
+#######################
+
+The tutorial describes the steps to run zephyr as User OS on ACRN hypervisor. We are using
+Kaby Lake-based NUC model NUC7i5DNH and you can try other platform ACRN supported as well.
+
+Introduction to Zephyr
+**********************
+
+The Zephyr is a scalable real-time operating-system (RTOS) supporting multiple hardware architecture,
+optimized for resource constrained devices, and built with safety and security in mind.
+
+Steps for Using Zephyr as User OS
+*********************************
+
+#. Build Zephyr and grub2 boot loader image
+
+   First of all, please follow the `Development Environment Setup on Linux
+   <https://docs.zephyrproject.org/latest/getting_started/installation_linux.html>`_ to
+   setup Zephyr development environment. Then follow the `Board-Specfic Support
+   <https://docs.zephyrproject.org/latest/boards/index.html>`_ to build the acrn supported zephyr image ``zephyr.elf``.
+
+   Build grub2 boot loader image
+
+   .. code-block:: none
+
+      $ git clone https://github.com/zephyrproject-rtos/zephyr.git
+      $ cd $ZEPHYR_BASE
```

Comment:
This does not make much sense in this sequence, `ZEPHYR_BASE` is not defined just after the `git clone`. I think the best is to refer to the Getting Started Guide and ask people to go through the various steps, such as installing the host tools, pick a Zephyr SDK, configure their `~/.zephyrrc`, ecc... and only mention the script below in this doc.

Reply:
Ok. Will do.